src/main/java/com/sublinks/sublinksapi/person/entities/Person.java

```diff
@@ -162,34 +162,31 @@ public class Person implements UserDetails, Principal {
   @Column(nullable = false)
   private String password;
 
-  @Column(nullable = false)
+  @Column(nullable = true, name = "avatar_image_url")
   private String avatarImageUrl;
 
-  @Column(nullable = false, name = "banner_image_url")
+  @Column(nullable = true, name = "banner_image_url")
   private String bannerImageUrl;
 
-  @Column(nullable = false)
+  @Column(nullable = true)
   private String biography;
 
-  @Column(nullable = false, name = "interface_language")
+  @Column(nullable = true, name = "interface_language")
   private String interfaceLanguage;
 
-  @Column(nullable = false, name = "default_theme")
+  @Column(nullable = true, name = "default_theme")
   private String defaultTheme;
 
-  @Column(nullable = false, name = "default_listing_type")
+  @Column(nullable = true, name = "default_listing_type")
```

Comment:
I don't believe this should be nullable.

Reply:
^